spacetemplate/assets/legacy.yaml

```diff
@@ -227,6 +249,8 @@ work_item_type_groups:
   - *featureID
   - *bugID
   bucket: requirement
+  description: Work item type group that focuses on execution-oriented work item
```

Comment:
For multi-line strings please use `>` and indentation as done in the space template's description.

Reply:
Fixed.